android/app/src/main/java/org/smartregister/fhircore/data/SamplePatients.kt

```diff
@@ -70,6 +70,8 @@ class SamplePatients {
     return patients
   }
 
+  fun getPatientItem(fhirPatient: Patient?) = fhirPatient?.let { createPatientItem(1, it) }
```

Comment:
is it ok to pass `id = 1` for every patient here? @Rkareko

Reply:
It's not an `id` it's a `position` that is currently not in use

Reply:
@shoaibmushtaq25 This function has the same result as passing a list of 1 element to the function below. The id is a nonnull field 

```
 fun getPatientItems(fhirPatients: List<Patient>): List<PatientListViewModel.PatientItem> {
    val patients: MutableList<PatientListViewModel.PatientItem> = mutableListOf()

    // Create a list of PatientItems from fhirPatients. The display index is 1 based.
    fhirPatients
      .take(MAX_RESOURCE_COUNT)
      .mapIndexed { index, fhirPatient -> createPatientItem(index + 1, fhirPatient) }
      .let { patients.addAll(it) }

    // Return a cloned List
    return patients
  }
```

Ideally, we should remove the class `SamplePatients`

Reply:
got it, Thanks for the detailed answer. @Rkareko @syedowaisali90